ykcapi/yk.h

```diff
@@ -48,4 +48,43 @@ YkLocation yk_location_new(void);
 // will occur.
 void yk_location_drop(YkLocation);
 
+void yk_stopgap(void *addr, uintptr_t size, uintptr_t retaddr, void *rsp);
+
+void __llvm_deoptimize(void* addr, uintptr_t size) {
```

Comment:
Should this be `#[cfg(target_arch = "x86_64")]`?

Reply:
Fixed in 0a14653.